flink-libraries/flink-table/src/main/scala/org/apache/flink/table/codegen/CodeGenerator.scala

```diff
@@ -582,13 +562,14 @@ class CodeGenerator(
          |
          """.stripMargin
 
-    funcCode += genSetAggregationResults(accTypes, aggs, aggMapping) + "\n"
+    funcCode += genSetAggregationResults(accTypes, aggs, aggMapping, partialResults) + "\n"
```

Comment:
good point. We can do it either way. I don't have a preference here. What do you prefer?